fastlane_core/lib/fastlane_core/feature/feature_manager.rb

```diff
@@ -0,0 +1,53 @@
+module FastlaneCore
+  class FeatureManager
+    class << self
+      attr_accessor :enabled_features
+    end
+    @enabled_features = []
+
+    def self.experiments_enabled?
+      return ENV['FASTLANE_ENABLE_ALL_EXPERIMENTS']
+    end
+
+    def self.enabled?(key)
+      feature = features.detect { |feat| feat.key == key }
+      return false if feature.nil?
+      return true if experiments_enabled? && feature.experiment == true
```

Comment:
Nit: no need to compare `feature.experiment` to `true` if it is already a boolean.

Additionally, consider getting rid of the `attr_accessor` for `experiment` on `Feature` and replace it with a method called `experiment?` which would be slightly more Ruby-like for a boolean-returning method.